src/Sarif.Viewer.VisualStudio.Interop/Sarif.Viewer.VisualStudio.Interop.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup Label="AssemblyAttributes">
     <AssemblyTitle>SARIF Viewer Interop Library</AssemblyTitle>
-    <Description>Provides an interop API that enables Visual Studio extensions to consume services provided by the SARIF Viewer extension.</Description>
+    <Description>Provides a simple API that allows Visual Studio extensions to easily consume services exposed by the Microsoft SARIF Viewer extension.</Description>
```

Comment:
>simple [](start = 28, length = 6)

"simple", "easily"... Well, I suppose there's nothing wrong with a bit of marketing. If it's me, I omit both those words. This is a public-facing string, so perhaps @michaelcfanning has an opinion.

Reply:
Fair enough, will dry it out :)

---
In reply to: [217543599](https://github.com/Microsoft/sarif-visualstudio-extension/pull/34#discussion_r217543599) [](ancestors = 217543599)